pack/TSC.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
+        "code": "11011",
+        "cost": null,
```

Comment:
As discussed in #241, using a `null` cost will cause Bridge of Dreams to display as an 'X' cost card instead of a '-' card. The data schema should be updated to distinguish between 'X' and '-' as two separate costs, which #242 does by updating cost to always be a string.